src/main/resources/org/jenkinsci/plugins/appdetector/AppDetectorBuildWrapper/config.jelly

```diff
@@ -2,20 +2,18 @@
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <f:block>
     <link href="${resURL}/plugin/app-detector/css/combo-box.css" rel="stylesheet" type="text/css" />
-    <table style="margin-left:2em">
```

Comment:
This `table` tag seems to interfere with `f:block`. 
Inner contents have changed only in indentation.